odoo/addons/base/models/ir_model.py

```diff
@@ -2185,6 +2188,26 @@ def toggle_noupdate(self, model, res_id):
             for xid in  self.search([('model', '=', model), ('res_id', '=', res_id)]):
                 xid.noupdate = not xid.noupdate
 
+    @api.autovacuum
+    def _gc__export__xml_ids(self):
+        """Garbage collection in ir.model.data models only for xml_id '__export__'
+        These kind of records could be too much during the time and many of them are useless after a while
+         - Just deleting records that are not used anymore after a while (VACUUM__EXPORT__MAX_WEEKS)
+         - Just deleting a few records each time (VACUUM__EXPORT__MAX_RECORDS)
+           in order to avoid raising an error if the worker timeout configuration is very low
+        """
+        limit_date = datetime.datetime.utcnow() - datetime.timedelta(weeks=VACUUM__EXPORT__MAX_WEEKS)
+
+        records = self.env['ir.model.data'].search([
+            ('module', '=', '__export__'), ('noupdate', '=', False),
+            ('create_date', '<', limit_date), ('write_date', '<', limit_date)],
```

Comment:
Why checking the creation date?

Reply:
It was inspired by 
 - https://github.com/odoo/odoo/blob/c5d3a109f530cd0763a1c3860086b5b7abef1b64/addons/mail/wizard/mail_compose_message.py#L533-L534

`write_date` is enough for me, maybe a double check with `create_date` is better for Odoo dev team